### PR TITLE
Move from promise-mysql to squelize

### PR DESCRIPTION
**Pull request by McInkay**
This will allow us to plug in sqlite in the next PR.

This should obviously require full testing again.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Note: we recommend yarn due to it's improved caching and more reliable dependenc
 * `yarn` or `npm install`
 * `mv config.default.js config.js`
 * edit config.js and update information
+* If using database, add moderators into database
```
Comment (Raia) [inline README.md]: Might not necessarily always be moderators?

```suggestion
* If using database, add users into database
```

**Comment by McInkay** [comment]
Yeah, should be. Sorry, I've been a bit busy. But it should be good to test.

**Comment by robdy** [comment]
LGTM, tested message relying and setting timezone.

**Comment by McInkay** [comment]
Nudge. @Raia you got any time to look at this? I'd love to get this in asap so I can move on to the final sqlite PR and move on to more interesting things.

**Comment by McInkay** [comment]
@Raia says she is good with just having @robdy 's review, as she doesn't have time. So I'll just merge.

**Pull request merged**